### ISSUE TITLE
Tools

### ISSUE THREAD

**Issue by CruzBishop** [Techern/Igneous-Extras#11]
- [x] Swords
- [x] Axes
- [x] Pickaxes
- [x] Shovels
- [x] Hoes (don't invite House)
- [ ] Textures
- [ ] Pirate translation
- [ ] Dutch translation
- [x] Config (damage value, durability, enchantibility, etc.)

**Comment by HoldYourWaffle** [comment]
I'm willing to do this for you if you want :)
But maybe it's better for me to wait till the 1.9 version is "done"

**Comment by CruzBishop** [comment]
If you're willing, it would help out a lot :) Even if it's a WIP when you open up a review, it can be expanded on (hopefully easily, it's been a while since I've done this :disappointed:)

(And thank you again, you're awesome)

**Comment by HoldYourWaffle** [comment]
Is 0.0.2 going to be 1.9?

**Comment by CruzBishop** [comment]
Just pulled 1.9's changes in b86f1ee0694f39c6a9d5945da955f0b8838b67c7, looks like it will be now :+1: 

It broke a lot less than I thought it would, which was nice :)

**Comment by HoldYourWaffle** [comment]
Ah nice :)
I'll get to this at sometime, so you can concentrate at other awesome stuff for this mod :+1:

**Comment by CruzBishop** [comment]
Thank you :+1: 

I have an early shift at work tomorrow so I might grab some sleep now and try to sneak some changes in before I go in the morning, otherwise I'll be back tomorrow night :)

**Comment by HoldYourWaffle** [comment]
Funny cause for you it's now around 11PM (if my time-difference calculations are correct [cough internet]) but for me it's 2PM :)

**Comment by CruzBishop** [comment]
10 PM, you were so close :p I hope you have a nice afternoon, Ravi - These last few days have been fun. I haven't had fun for a while, so thank you :+1:

**Comment by HoldYourWaffle** [comment]
Np m8 =)

(I think my calculations messed up with the fact that the site reported +9 when I selected Amsterdam (NL) so I thought Austalia had +10 cause NL has already +1 but that wasn't the fact appearently)

**Comment by HoldYourWaffle** [comment]
I expect to have atleast a WIP of the tools on wednesday (6/4) ready

Oh btw don't expect much from the textures.

(I'm still in school so haven't got all time :-1: )

Edit: got excursion tomorrow (5/4) and test the day after; delaying till 8/4 or at the very least 9/4

**Comment by HoldYourWaffle** [comment]
PR #20

**Comment by CruzBishop** [comment]
Awesome, thanks :) I'll go check it out

**Comment by HoldYourWaffle** [comment]
Nice now I got my checklist =)

**Comment by CruzBishop** [comment]
Congratulations, @ShadowCoder69 - Your first assigned issue :+1: 

<sup><sup><sup>Good luck :D</sup></sup></sup>

**Comment by CruzBishop** [comment]
Hey @ShadowCoder69 are you on Windows, Mac or a Linux distribution? Just checking in my coffee break so you'll get a chance to answer before I get home :-)

**Comment by HoldYourWaffle** [comment]
(Just woke up)
Currently on my phone (android), but my pc has windows 10

**Comment by HoldYourWaffle** [comment]
Shoudn't IABlocks be named IEBlocks?

**Comment by HoldYourWaffle** [comment]
I added a ConfigHandler in e49cadd, do you like it?

**Comment by CruzBishop** [comment]
Alright, thanks, and welcome back :p

---

Oops. Yeah, sorry -  I'll change that over in a sec

---

ConfigHandler looks great, thank you :+1:

**Comment by HoldYourWaffle** [comment]
Next commit will polish ConfigHandler a bit and acutually register the tools

Btw it is so damn painfull to see all that registers in IA/EBlocks and knowing they need to be replaced
(#24)

**Comment by HoldYourWaffle** [comment]
Renamed the IABlocks (d8b7b7e)

**Comment by HoldYourWaffle** [comment]
Still zombie, but I'll try to work on this for a while

**Comment by CruzBishop** [comment]
Pretty much only translations and textures now :) Should be code-complete

(I'll try to do some textures on Monday :D)

**Comment by HoldYourWaffle** [comment]
Tonight I'm going to France (cause we're dutch people. And dutch people go to France. Mostly.), so I won't be doing very much for 2 weeks

**Comment by CruzBishop** [comment]
:fr: :free: :frog: :poultry_leg: :mask: :phone: :ambulance: :hospital: 

Enjoy your holiday, mate :D

**Comment by HoldYourWaffle** [comment]
Thx.

Btw, a really random question, you celebrate christmas in the summer right?

**Comment by CruzBishop** [comment]
Yep :( It's way too humid then but hey, no (major) earthquakes ;D

**Comment by HoldYourWaffle** [comment]
So it's winter there right now...

**Comment by CruzBishop** [comment]
Doesn't really feel like it right now, but yeah :) Feels like about 28 degrees because of the humidity (It rained for maybe an hour a little while ago)

Celsius: 

![celc](https://cloud.githubusercontent.com/assets/639417/17057255/41de8e3a-505c-11e6-894d-7e364becb292.PNG)

Fahrenheit: 
![fark](https://cloud.githubusercontent.com/assets/639417/17057260/4994bbb8-505c-11e6-9bc4-106289d8f6a5.PNG)

**Comment by CruzBishop** [comment]
Huh, I never noticed it changed km/h to m/h. Nice

**Comment by HoldYourWaffle** [comment]
Still any interest in this mod?

Btw it's really funny to read this conversation. I was just thinking about the whole australian christmas thingy a couple of days ago 😂  Also it's weird to see you say "22°C doesn't feel like winter", in the Netherlands that's about the average summer temperature (maybe a bit colder), anything above 25°C and we just melt miserably in our home 🌞
